projects/demo/src/modules/app/pages.ts

```diff
@@ -1396,6 +1396,13 @@ export const pages: TuiDocPages = [
                     'верстка, markup, скелетон, loader, загрузка, skeleton, shimmer',
                 route: 'directives/skeleton',
             },
+            {
+                section: 'Tools',
+                title: 'Amount',
+                keywords:
+                    'amount, money, деньги, сумма, количество, cash, копейки, рубли, доллары, евро',
+                route: '/pipes/amount',
+            },
```

Comment:
no, move this changes in array "subPages" of "section: 'Tools', title: 'Pipes'" and remove old one

Reply:
NO!!! 
1. Find object with "section: 'Pipes' title: 'Amount'"
2. move this object in array "subPages" of "section: 'Tools', title: 'Pipes'"  
3. Remove first object